src/views/communities/CommunityCard.tsx

```diff
@@ -30,184 +25,248 @@ function CommunityCard(props: { community: CommunityAttributes }) {
+                            <Text style={styles.cardCommunityTagText}>
+                                {setTextSuspects(Number(community.suspect))}
+                            </Text>
+                        </View>
+                    )} */}
                     <View style={styles.darkerBackground} />
                 </View>
-                <View style={{ marginHorizontal: 17, marginBottom: 16.24 }}>
+                <View
+                    style={{
+                        marginVertical: 16,
+                        height: 210,
+                        width: 254,
+                        justifyContent: 'space-between',
+                    }}
+                >
+                    <Text style={styles.cardCommunityName}>
+                        {community.name}
+                    </Text>
+                    <Text style={styles.cardCommunityDescription}>
+                        {community.description?.length > 100
+                            ? community.description.substr(0, 100) + '...'
+                            : community.description}
+                    </Text>
                     <View
                         style={{
-                            flex: 3,
                             flexDirection: 'row',
-                            marginTop: 19.27,
-                            marginBottom: 17.85,
+                            alignItems: 'center',
+                            bottom: 0,
                         }}
                     >
-                        <View style={{ flex: 1 }}>
-                            <Text style={styles.cellHeader}>
+                        <View
+                            style={{
+                                alignItems: 'center',
+                                justifyContent: 'center',
+                                flexDirection: 'row',
+                                marginRight: 12,
+                            }}
+                        >
+                            <BeneficiariesSvg style={{ marginRight: 4 }} />
+                            <Text style={styles.bottomCardText}>
                                 {community.state.beneficiaries}
                             </Text>
-                            <Text style={styles.cellDescription}>
-                                {i18n.t('generic.beneficiaries')}
-                            </Text>
                         </View>
-                        <View style={{ flex: 1 }}>
-                            <Text style={styles.cellHeader}>
-                                $
-                                {humanifyCurrencyAmount(
-                                    community.contract.claimAmount
-                                )}
-                            </Text>
-                            <Text style={styles.cellDescription}>
-                                {claimFrequencyToText(
-                                    community.contract.baseInterval
+                        <View
```

Comment:
Just by looking at the code, I know there's one thing missing here.

Reply:
Sincerely I couldn't  find what it is after reading over and over again. UI wise seems to match Figma. UX too. So, I need you to tell me what's you found missing.